### ISSUE TITLE
JWT expiry

### ISSUE THREAD

**Issue by mffap** [caos/zitadel#1450]
**Describe the bug**

- [Documentation](https://docs.zitadel.ch/architecture#JSON_Web_Token_JWT_Profile) states that `exp` claim must not be longer than 1h. 
- I was able to create jwt with `exp` > 60min without error
- Not sure if it is possible to request tokens after 60min, though

**To Reproduce**
1. Create JWT with exp 120min and sign with private key
2. Request access token from /oath/v2/token
3. Success

**Expected behavior**
- Receive an error that `exp` must be less than an hour
- ZITADEL should validate  `exp` of the assertion

**Additional context**

- Make sure to update documentation ("architecture" and "guides")

**Comment by livio-a** [comment]
might need a change in the oidc lib

**Comment by livio-a** [comment]
I finally checked this. We validate `iat` to be not more than 1h ago and not `exp` to be less than 1h.

I think this is ok and we should just change the docs.

Btw. the link above is no longer up to date. Sections I've found which would have to be changed:
- https://docs.zitadel.ch/docs/apis/openidoauth/authn-methods/#jwt-with-private-key
- https://docs.zitadel.ch/docs/guides/authentication/serviceusers#2-create-a-jwt-and-sign-with-private-key
- more?
